### PR TITLE
Change the content column in the spotlight_pages table from…

### PR DESCRIPTION
**Pull request by jkeck**
…text to mediumtext.

### CODE REVIEW

File: db/migrate/20180529225807_change_spotlight_pages_content_column_to_medium_text.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ChangeSpotlightPagesContentColumnToMediumText < ActiveRecord::Migration[5.2]
```
Comment (cbeer) [inline db/migrate/20180529225807_change_spotlight_pages_content_column_to_medium_text.rb]: Just one stupid question; will specifying `[5.2]` here mean that Rails 5.1 won't be able to run the migration?

Comment (cbeer) [reply db/migrate/20180529225807_change_spotlight_pages_content_column_to_medium_text.rb]: Answer: yes; Rails refuses to run a migration that's tagged with a greater version. I think we need to specify `5.1` instead.

**Comment by jkeck** [comment]
👍

File: db/migrate/20180529225807_change_spotlight_pages_content_column_to_medium_text.rb (hunk 1 above)
Comment (jkeck) [reply db/migrate/20180529225807_change_spotlight_pages_content_column_to_medium_text.rb]: Yeah, I just did whatever rails generating for me w/o thinking too much about it. 😳

**Pull request merged**